### ISSUE TITLE
give us the ability to turn off sustained note keyboard display

### ISSUE THREAD

**Issue by Nwachinyere** [matkatmusic/ChordieApp_IssueTracker#58]
Chordie 1.1.2 had a feature that allowed turning off sustained notes and overlay for keyboard view.  There is an option in version 1.2.3 to turn off "enable sustain overlay" but it still displays sustained notes.  Please allow the us to turn off sustained notes as well.  it's hard to follow which notes are sustained and which ones are being played just with the keyboard view.

**Comment by matkatmusic** [comment]
Did you reset your settings file before testing? @Nwachinyere

**Comment by matkatmusic** [comment]
I'm not sure what you're experiencing, but the sustain toggle works as expected.

https://www.dropbox.com/s/r6j9rqbkhl959nu/Sustain.mp4?dl=0

**Comment by Nwachinyere** [comment]
I made sure to reset settings before I tested. Maybe you didn't understand
the request so I'll try to re-explain.  When overlay is off, the sustained
notes are still displayed.  The only thing that toggle does is turn off the
overlay color so that sustained notes and notes being played are the same
color. I've included pictures of this behavior below. Both versions have
the overlay turned off and those are notes that I've held with the sustain
pedal but chordie 1.2.3 still displays the sustained notes on the keyboard.
I don't know if this is intentional, but I can say that it makes it more
difficult for my students to follow what I'm actually playing. I use the
sustain pedal a lot when I play.

[image: Chordie 1.2.3.png]
[image: chordie 1.1.2.png]

**Comment by Nwachinyere** [comment]
https://drive.google.com/file/d/1-qlAskFhjvgKehHW1qagDER7zc4efxUv/view?usp=sharing
https://drive.google.com/file/d/1gcDwv5jDYGqpwykMAm7c5bDPdtgsya7v/view?usp=sharing

**Comment by Nwachinyere** [comment]
@matkatmusic  please see comments above

**Comment by matkatmusic** [comment]
If notes are sustaining on your controller because you are holding the sustain pedal down then Chordie is going to display them as "on".  Maybe lighten up on your sustain pedal usage?

**Comment by Nwachinyere** [comment]
It's understood that this is not an issue or a bug in the software.  In fact, it may have been a bug in the old code that caused it to behave like that. It would be nice if it was an added feature for people who preferred the way the software used to work.  I don't mean to come off as if I don't appreciate all the work that was done in this new update. I know there was a lot of coding behind the new changes. This update solved all the other issues I had with the old software. I guess I'll just adjust to the way it works now.

**Comment by matkatmusic** [comment]
Thanks.  I've had other people wonder about the sustain feature before.  They were like "notes aren't releasing".  Turns out they were literally resting their foot on the sustain pedal the whole time and never releasing it.  Once I pointed it out, they realized that they had bad pedal technique and quickly worked to correct a very bad playing habit
